### ISSUE TITLE
Errors using mcxlab or mcx with vol.bin files

### ISSUE THREAD

**Issue by kdreher** [fangq/mcx#67]
In the mcx-users google group [answer](https://groups.google.com/d/msg/mcx-users/OleiBgSzG3E/hQ0x14QlBAAJ), I attached some scripts that should reproduce the errors as well as one script that should fix the mcxlab error.

First use 'test_script.m' to convert a cfg to a json and reproduce the mcxlab error.
Then, comment the line 36 in 'test_script.m' and run it again to use a modified version of 'json2mcx.m' that fixes this error.

In order to reproduce the mcx error,  the 'test.sh' shell script has to be executed (with the mcx executable in the same directory) with the json file as input, that was created in the previous step by running 'test_script.m'.

**Comment by kdreher** [comment]
Concerning the mcx error: 
I looked into the code and printed the cfg->mediabyte values, that are actually used, while specifying different values in the command line. I found out that the mediabyte value is somehow not read correctly, since for every possible value for --mediabyte, it always printed the default value of 1. 

Additionally, by trying different default values, for example 100, the simulation ran without error but the mc2 file contained a volume with wrong dimensions. So even if the mediabyte value is right, there is another problem which I haven't figured out yet.
